extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -79,6 +80,17 @@ private YoutubeParsingHelper() {
     private static final String[] HARDCODED_YOUTUBE_MUSIC_KEYS = {"AIzaSyC9XL3ZjWddXya6X74dJoCTL-WEYFDNX30", "67", "0.1"};
     private static String[] youtubeMusicKeys;
 
+    /**
+     * <code>PENDING+999</code> means that the user did not yet submit their choices.
+     * Therefore, YouTube & Google should not track the user,
+     * because they did not give consent. The three digits at the end can be random.
+     */
+    public static final String CONSENT_COOKIE_VALUE = "PENDING+999";
```

Comment:
Randomize it a bit to have 100-999:

public static final String CONSENT_COOKIE_VALUE = "PENDING+" + (100 + new Random().nextInt(900));